data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +518,1020 @@ mission "Unfettered Jump Drive Trading"
+			`	As you move closer to inspect it you are almost frightened to death by the huge scorpion looking creature rushing towards you.`
+			`	There is little time to think as the huge creature gets closer and closer!`
+			choice
+				`	(Run far away in case it escapes!)`
+					goto run
+				`	(Leap back to be at a respectable distance!)`
+				`	(Stay there.)`
+					goto stay
+				`	(Shoot it.)`
+					goto shoot
+				`	(Try to calm it.)`
+					goto calm
+			`	Surprisingly, no Unfettered look amused by the apparent cowardliness. "Fear is good, so long as it does not control you," one of them says.`
+			label calm
+			action
+				"unfettered rep: Si-Ra" ++
+			`	You hold out your hand peacefully to try and appease it as it jumps out. Surprisingly, it stops right in front of you, and observes you, its eyes deep as the void. The Unfettered are all gawking at you, impressed by this feat of courage that is seemingly paying off. Then a nearby warrior points their weapons at it, and the scorpion-like creature moves faster than you thought possible, striking you with its tail. Then, everything gets dark.`
+				goto poisoned
+			label reward
+			action
+				outfit "Heshi"
+			`	The Captain does not give you any explanation, instead he proceeds to raise his fist. As you do not react he gestures with his other hand, clearly meaning you ought to do the same.`
+			`	After you raise your fist in response, he bows his head so slightly you barely notice it, and moves away. You wonder what you should do with the creature, but refusing the gift does not seem like an option.`
+			label end
+			action
+				log `Defeated an Unfettered Captain in a duel, and took part in a surprisingly scary Unfettered martial ceremony.`
+				event "unfettered: JD raid plan ready" 5 15
+				set "unfettered: duel ready"
+				# there's no way to get these before the duel
+				"unfettered: took Si-Ra' stuff" = "outfit (all): Gold Bars" + "outfit (all): Ionic Blaster Prototype" * 2
+			`	As he passes near you he quickly speaks, "It'll be a few weeks before I'm ready for more action."`
+				accept
+			label stay
+			`	You stay there, surely impressing all Hai present! However it also means you get hit by the tail of the monster as it jumps out to hit you, and everything gets dark.`
+			label poisoned
+			`	You hear voices around you, someone screaming for help, but also your recent opponent speaking calmly. "There is no cure for humans," you hear as you sink deeper and deeper, down to the edge of unconsciousness, "there is nothing to be done. This is a fight they will have to win on their own."`
+			`	Suddenly the world darkens, and you fall. When you open your eyes again, all is vivid, as if you had never seen before. You see the scorpion-like creature that attacked you, clearer than ever, in all its might and with a deep spark of intelligence in its eyes, before it swiftly moves away.`
```

Comment:
```suggestion
			`	Suddenly the world darkens, and you fall. When you open your eyes again, all is vivid, as if you had never seen before. You see the scorpion-like creature that attacked you, clearer than ever, in all it's might and with a deep spark of intelligence in it's eyes, before it swiftly moves away.`
```

Reply:
https://www.merriam-webster.com/grammar/when-to-use-its-vs-its